samples/kotlin-console/src/main/kotlin/org/jellyfin/sample/console/cli/Ping.kt

```diff
@@ -11,8 +14,14 @@ class Ping(
 	private val jellyfin: Jellyfin
 ) : Subcommand("ping", "Pings a given server and retrieve basic system information") {
 	private val server by option(ArgType.String, description = "Url of the server", shortName = "s").required()
+	private val extended by option(ArgType.Boolean, description = "Normalize the URL and request", shortName = "e").default(false)
```

Comment:
The description isn't that obvious in what this actually does, but since it's only a sample it doesn't really matter I guess.

Reply:
I think I forgot to change it when I copied another boolean from a different file